ios/PurchasesHybridCommon/Podfile

```diff
@@ -1,12 +1,13 @@
 # Uncomment the next line to define a global platform for your project
 
 target 'PurchasesHybridCommon' do
-  platform :ios, '9.0'
+  platform :ios, '11.0'
```

Comment:
took the opportunity to update the deployment targets. This simplifies code in a few places where we were checking for older OS versions. 11.0 matches purchases-ios v4